src/test/java/com/google/javascript/clutz/private_type.js

```diff
@@ -0,0 +1,7 @@
+goog.provide('private_type.user');
```

Comment:
nit: underscore namespaces are against Closure style guide (probably guilty of few of those in our tests myself).